plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java

```diff
@@ -2109,28 +2148,31 @@ public void registerCoreMembers() {
         // @Syntax leash (cancel) [<entity>|...] (holder:<entity>/<location>)
         // @Required 1
         // @Stable stable
-        // @Short Sticks a leash on target entity, held by a fence post or another entity.
+        // @Short Sticks a leash on target entity, held by a fence or another entity.
         // @Author Alain Blanquet, mcmonkey
         // @Group entity
         //
         // @Description
-        // TODO: Document Command Details
+        // Attaches a leash to the specified entity.
+        // The leash may be attached to a fence, or another entity.
+        // Players/NPCs may not be leashed.

```

Comment:
Can non-player type NPCs be leashed?

Reply:
Non-player NPCs are leashable specifically when `/npc leashable` is toggled to allow that - docs should mention that